src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -46,10 +46,15 @@ class OpenTelemetryHandle implements TracingHandle {
 
     private Tracer tracer;
 
+    @SuppressWarnings("deprecation")
     static void setCommonAttributes(SpanBuilder builder, RoutingContext routingContext) {
         builder.setAttribute(SemanticAttributes.PEER_SERVICE, KAFKA_SERVICE);
-        builder.setAttribute(SemanticAttributes.HTTP_METHOD, routingContext.request().method().name());
-        builder.setAttribute(SemanticAttributes.HTTP_URL, routingContext.request().uri());
+        builder.setAttribute(SemanticAttributes.HTTP_METHOD, routingContext.request().method().name()); // TODO remove in release after 0.28
+        builder.setAttribute(SemanticAttributes.HTTP_REQUEST_METHOD, routingContext.request().method().name());
+        builder.setAttribute(SemanticAttributes.HTTP_URL, routingContext.request().uri()); // TODO remove in release after 0.28
```

Comment:
The CHANGELOG suggests these are removed. Here you say remove after release 0.28. Why? If we keep them for backwards compatibility, that is great. But might make sense to give it few more releases. It should be also explained in the changelog properly what is being done.

Reply:
I've updated the changelog to better reflect the change and reworded this comment to more accurately reflect the intent.